airflow/www/views.py

```diff
@@ -837,8 +837,8 @@ def log(self):
         if ti:
             host = ti.hostname
             log_loaded = False
-
-            if socket.getfqdn() == host:
+            
+            if conf.get('webserver', 'FORCE_LOCAL_LOGS') or socket.getfqdn() == host:
```

Comment:
Yea, honestly, I'm wondering if there's some undocumented reason for this. I kind of doubt it, though.

@aoen @mistercrunch any reason why there isn't a simple os.path.exists check here?